### PR TITLE
Adding samples z-stream approvers

### PR DESCRIPTION
**Pull request by dperaza4dustbit**
Adding myself (primary) and Allen Bai (backup) as z-stream
approvers. Following instructions in the OpenShift Backport
process. Also adding Sushanta Das (primary) and Pavel Macik
(backup) as cherry-pick z-stream approvers. Got green light
from Pavel who is the QE Team Lead.

### CODE REVIEW

File: core-services/prow/02_config/openshift/cluster-samples-operator/_pluginconfig.yaml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+label:
+  restricted_labels:
+    openshift/cluster-samples-operator:
+    - allowed_users:
```
Comment (gabemontero) [inline core-services/prow/02_config/openshift/cluster-samples-operator/_pluginconfig.yaml:4]: I *think* you should have an 
``
  allowed_teams:
  - openshift-patch-managers
 ```
entry  preceding the `allowed_users` entry in the `openshift/cluster-samples-operator` array

I at least see that @dperaza4dustbit in the components @adambkaplan and @coreydaley set up for build-api team

Comment (gabemontero) [inline core-services/prow/02_config/openshift/cluster-samples-operator/_pluginconfig.yaml:4]: I *think* you should have an entry like https://github.com/openshift/release/blob/master/core-services/prow/02_config/openshift/openshift-controller-manager/_pluginconfig.yaml#L4-L5 and include something similar in your change here

@adambkaplan @coreydaley would you guys agree?

wdyt @dperaza4dustbit ?

Comment (adambkaplan) [reply core-services/prow/02_config/openshift/cluster-samples-operator/_pluginconfig.yaml:4]: yes, you need to allow the patch managers to add this label, too.

Comment (dperaza4dustbit) [reply core-services/prow/02_config/openshift/cluster-samples-operator/_pluginconfig.yaml:4]: OK, changing, thanks

**Comment by dperaza4dustbit** [comment]
FYI, adding QE Cherry-pick z-stream approver on behalf of our QE, discussed this and got an OK from @dhellmann here: https://coreos.slack.com/archives/CJARLA942/p1636464502245800?thread_ts=1636386952.236800&cid=CJARLA942

**Pull request merged**